neurom/features/neuronfunc.py

```diff
@@ -209,24 +221,25 @@ def _sort_angle(p1, p2):
             for i, _ in enumerate(ordered_vectors)]
 
 
-def sholl_crossings(neurites, center, radii, neurite_filter=None):
+def _sholl_crossings(neurites, center, radii, neurite_type=NeuriteType.all):
```

Comment:
@arnaudon I've made this function private. Is it ok for you? Did you use it?

Reply:
ah could you leave it public, we could use it with specific argument for neurite type if recall, thanks!